deploy/kcp-hacbs-workspace-init/templates/userconfig.yaml

```diff
@@ -0,0 +1,36 @@
+#TODO  long term, do we want to merge this with <repoHome>/deploy/base/config.yaml ??
```

Comment:
@stuartwdouglas @dwalluck I wanted to highlight this

we probably should reach a consensus on where the master/all possible options we want to expose lives

For example, in @dwalluck 's latest PR https://github.com/redhat-appstudio/jvm-build-service/pull/328/files#diff-323f42935c5af98a0b4152788343529680bef21786b46d6d23f8c23c6d785755R198 where he is updating the e2e's set up of the user config to add more kotlin locations, it may be time to work off of yaml vs. golang for stuff like that?

WDYT?

Reply:
over time @stuartwdouglas exposing any of these fields as template parameters is what I was trying to describe during our sync up last night

Reply:
A lot of these repos are really required for rebuilding dependencies, rather than something end users will want. I realised recently as this list has grown that a lot of these they should actually be a property of the build recipe repo rather than the users workspace. 

I am not actually sure how to do that yet though, in a lot of ways it would have been easier when we had the sidecar.